exp/services/recoverysigner/internal/serve/serve.go

```diff
@@ -99,6 +105,11 @@ func getHandlerDeps(opts Options) (handlerDeps, error) {
 	}
 	opts.Logger.Infof("SEP10 JWKS contains %d keys", len(sep10JWKS.Keys))
 
+	encrypter, decrypter, err := crypto.NewEncrypterDecrypter(opts.EncryptionKMSKeyURI, opts.EncryptionTinkKeysetJSON)
+	if err != nil {
+		return handlerDeps{}, errors.Wrap(err, "error initializing EncrypterDecrypter")
```

Comment:
You don't need to say its an error. If every wrap did that it'd say error repeatedly:
```suggestion
		return handlerDeps{}, errors.Wrap(err, "initializing encrypter decrypter")
```

Reply:
I was following the pattern in the file. We might want to remove `error` from other messages used in `Wrap` in this file then.

Reply:
https://github.com/stellar/go/pull/2692/commits/aba4fb411f987351acfc03ab76897a842f1ffd15